server/publicapi/items/service.py

```diff
@@ -228,6 +241,129 @@ def _create_date_range_filter(self, start_date, end_date):
 
         return date_filter
 
+    def _set_fields_filter(self, req):
```

Comment:
They are meant as helper methods to make the "public" methods more readable. An implementation detail basically, they are not designed to be general-purpose utility methods.

The reason is partially because the `PackagesService` already shares the code with the `ItemsService` via inheritance. On the other hand, if we had multiple independent services, then it would indeed be a good idea to put these helper methods (e.g. parameter parsing) into some common utility class.